docs/source/contributing.rst

```diff
@@ -77,11 +77,15 @@ Installing a Development Version
    ::
 
       pip install -r requirements.txt  # Installs basic requirements
+      pip install -r dev_requirements.txt  # Installs developer requirements
       pip install -r docs/docs_requirements.txt # Installs documentation requirements
-      pip install -r examples/examples_requirements.txt # Installs example requirements
       pip install -e .  # Installs SCICO from the current directory in editable mode
 
 
+   For installing dependencies related to the examples please see :ref:`example_notebooks`.
+   Installing these are neccessary for the successfull running of the tests.
```

Comment:
This seems easy to miss to me, but I think it's fine for now given that we will likely look at all of this again for #155 .